libultraship/libultraship/SDLController.cpp

```diff
@@ -440,6 +440,8 @@ namespace Ship {
         profile->Mappings[SDL_CONTROLLER_BUTTON_START] = BTN_START;
         profile->Mappings[SDL_CONTROLLER_BUTTON_B] = BTN_B;
         profile->Mappings[SDL_CONTROLLER_BUTTON_A] = BTN_A;
+        profile->Mappings[SDL_CONTROLLER_BUTTON_LEFTSTICK] = BTN_MODIFIER1;
+        profile->Mappings[SDL_CONTROLLER_BUTTON_RIGHTSTICK] = BTN_MODIFIER2;

```

Comment:
might be nice to add a comment explaining why the modifiers are being mapped to the sticks

Reply:
Funny enough, the real reason isn't even relevant anymore!  Back when I first made this PR, the C buttons had that commented out double-mapping that also took up the X, Y, Right Shoulder, and Select buttons (I believe), so the two stick buttons were the only ones left unmapped!  The default mapping has since removed that entirely, but because people might still be using some version of that mapping (hint hint ME, lol), I didn't want to potentially conflict with them.  I'm curious if you think people will be more likely to already have something mapped to the stick buttons though, since I know I've seen some people put Up C on right stick click.  

Plus, in my testing, it actually ended up being kind of useful to just click and then aim the left stick with a single finger, kind of like Metroid Dread ended up doing, and I've never felt like the imprecision of the stick buttons lent themselves well to use as actual items.